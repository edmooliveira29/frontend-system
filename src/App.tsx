import { router } from './router/router'

function App() { return router }

export default App
