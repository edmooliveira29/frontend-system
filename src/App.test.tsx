import React from 'react';
import {render, screen} from '@testing-library/react';

import App from './App';

test('Should be on the main page', () => {
  render(<App />);
  const linkElement = screen.getByText('Home');
  expect(linkElement).toMatchSnapshot('<h1>Home</h1>');
});
