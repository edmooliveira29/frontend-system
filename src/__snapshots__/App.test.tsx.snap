// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should be on the main page: <h1>Home</h1> 1`] = `
<h1>
  Home
</h1>
`;
